read_motif_v6.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Please only add a _single_ docstring. Also, please omit the author information, this is automatically tracked by Git, on a line-per-line basis. Adding this here will lead to artifacts, because these metadata are never properly updated when maintaining the code

Reply:
Please have a look at and follow the style of Google-style docstrings: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html